projects/fast-xml-parser/xml.toml

```diff
@@ -0,0 +1,5 @@
+exit-on-time = 3600
+
+[jazzer_js]
+path = "/jazzer.js/fast-xml-parser/fuzz.js"
+args = "/corpus -- -workers=2 -jobs=100"
```

Comment:
Maybe add `-i` flag with path to the lib directory

Reply:
By default `-i` is set to * (instrument all). Although `--sync` must be specified for this target